tests/test_call_policies.cpp

```diff
@@ -0,0 +1,92 @@
+/*
+    tests/test_keep_alive.cpp -- keep_alive modifier (pybind11's version
+    of Boost.Python's with_custodian_and_ward / with_custodian_and_ward_postcall)
```

Comment:
The comment filename and description are obsolete now.  Perhaps:

    tests/test_call_policies.cpp -- keep_alive (pybind11's version of Boost.Python's
    with_custodian_and_ward / with_custodian_and_ward_postcall) and call_guard
    examples and tests.